app/jenkins-for-jira-ui/src/components/ConnectJenkins/ConfigurationSteps/ConfigurationSteps.test.tsx

```diff
@@ -3,18 +3,47 @@ import { render, screen } from '@testing-library/react';
 
 import { ConfigurationSteps } from './ConfigurationSteps';
 
+// Note: Using getByText() in this test as AtlasKit does not give you the ability to attach a testId to the elements.
+
 describe('ConfigurationSteps Page Suite', () => {
-	it('Should render two stages', () => {
+	it('Should render three stages', () => {
 		render(<ConfigurationSteps currentStage="install" />);
 		expect(screen.getByText('Install plugin')).toBeInTheDocument();
-		expect(screen.getByText('Connect your app')).toBeInTheDocument();
+		expect(screen.getByText('Create server')).toBeInTheDocument();
+		expect(screen.getByText('Connect server')).toBeInTheDocument();
 	});
 
 	it('Should render correctly with install as currentStage', () => {
 		render(<ConfigurationSteps currentStage="install" />);
 		const InstallJenkinsStep = screen.getByText('Install plugin');
-		const ConnectJenkinsStep = screen.getByText('Connect your app');
+		const CreateServerStep = screen.getByText('Create server');
+		const ConnectServerStep = screen.getByText('Connect server');
 		expect(InstallJenkinsStep).toHaveAttribute('style', 'color: rgb(0, 101, 255); font-weight: 600;');
-		expect(ConnectJenkinsStep).toHaveAttribute('style', 'color: rgb(94, 108, 132); font-weight: 400;');
+		expect(CreateServerStep).toHaveAttribute('style', 'color: rgb(94, 108, 132); font-weight: 400;');
+		expect(ConnectServerStep).toHaveAttribute('style', 'color: rgb(94, 108, 132); font-weight: 400;');
+	});
+
+	it('Should render correctly with create as currentStage', () => {
+		render(<ConfigurationSteps currentStage="create" />);
+		const InstallJenkinsStepParentContainer = screen.getByText('Install plugin').closest('div');
+		const CreateServerStep = screen.getByText('Create server');
+		const ConnectServerStep = screen.getByText('Connect server');
+
+		// Visited steps are styled via parent container
+		expect(InstallJenkinsStepParentContainer).toHaveAttribute('style', 'color: rgb(23, 43, 77); font-weight: 600;');
+		expect(CreateServerStep).toHaveAttribute('style', 'color: rgb(0, 101, 255); font-weight: 600;');
+		expect(ConnectServerStep).toHaveAttribute('style', 'color: rgb(94, 108, 132); font-weight: 400;');
+	});
```

Comment:
What's the reason we're testing the style?

Reply:
Conversation summary from slack:
The AtlasKit component doesn't render any other attributes to test on e.g. classes. The classes it does render are auto-generated and aren't reliable to test with.